components/zcash_protocol/src/value.rs

```diff
@@ -298,6 +317,15 @@ impl Zatoshis {
     pub fn is_positive(&self) -> bool {
         self > &Zatoshis::ZERO
     }
+
+    /// Divides this `Zatoshis` value by the given divisor and returns the quotient and remainder.
+    pub fn div_with_remainder(&self, divisor: NonZeroUsize) -> DivRem<Zatoshis> {
+        let divisor = u64::try_from(usize::from(divisor)).expect("divisor fits into a u64");
```

Comment:
This makes the code in `zcash_client_backend` a bit more annoying, because there we're dealing mostly with `usize` values that are coming from `.len()` calls; `NonZeroUsize` still makes sense for `target_output_count` in the `SplitPolicy` type I think?